docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # ----- Go Builder Image ------
 #
-FROM --platform=$BUILDPLATFORM golang:1.21 AS builder
+FROM golang:1.21 AS builder
```

Comment:
Please, keep the `BUILDPLATFORM`

Reply:
Reverted these changes though I guess the 2nd change in 63 shouldn't have to be reverted.

```text
 - RedundantTargetPlatform: Setting platform to predefined $TARGETPLATFORM in FROM is redundant as this is the default behavior (line 63)
 ```